tests/test.py

```diff
@@ -2341,6 +2343,22 @@ def get_base_translator_args(self):
         }
 
 
+class KeystoreCacheOnStartMixin:
```

Comment:
will be great to comment what this mixin do. because for me it wasn't obvious that it start acra-server and drops keystore folder)